android/build_androidtc_arm.sh

```diff
@@ -10,6 +10,6 @@ export NDK_ROOT="${NDK_ROOT:-$HOME/NVPACK/android-ndk-r10e}"
 export ANDROID_TC="${ANDROID_TC:-$HOME/android-tc}"
 export ANDROID_TC_ARM="${ANDROID_TC_ARM:-$ANDROID_TC/arm}"
 
-if [ ! -d $ANDROID_TC_ARM/arm-linux-androideabi ]; then
-    $NDK_ROOT/build/tools/make-standalone-toolchain.sh --platform=android-9 --arch=arm --install-dir=$ANDROID_TC_ARM
+if [ ! -d "$ANDROID_TC_ARM/arm-linux-androideabi" ]; then
```

Comment:
One could argue that this is redundant but, ok, it's future-proof should line 11 change or be removed.